src/DroneSafetyApi/Models/Heatmap.cs

```diff
@@ -1,12 +1,81 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
 
 namespace DroneSafetyApi.Models
 {
-    public class Heatmap
+
+    public class HeatMap
```

Comment:
I agree only that the generation of heatmap points and json converter should be somewhere else, the others are part of the model and would lead to annoying code otherwise